controller/system_rollout_controller.go

```diff
@@ -2036,7 +2036,7 @@ func (c *SystemRolloutController) tagLonghornLastSystemRestoreAnnotation(obj run
 		annos[types.GetLastSystemRestoreAtLabelKey()] = c.systemRestoredAt
 	}
 
-	log.Debug(message)
+	log.Info(message)
```

Comment:
The number of this log could be a lot if there are many resources to be restored. I would suggest debug log instead. WDYT? because from the debug purpose, we still can check the support bundle to see the restored resources have this annotation or not, so not really require this log somehow.

Reply:
Updated. Thank you.